Dynatrace/dynatrace-operator
# Fixes csi-driver stuck in Terminate after restart

# Description
The CSI-driver can get stuck if it mounted volumes recently and then got restarted.

This happens due to some unexpected behaviour when using the same volume for 2 volumeMounts in a container while having `mountPropagation` on both volumeMounts.

Also fixes a misleading error that happens on restart.

## How can this be tested?
Deploy a Dynakube that uses the CSI-driver, deploy sample apps. 
After they are done deploying, restart the CSI-driver,  it should restart without problem.


## Checklist
